tests/periph_rtt_min/README.md

```diff
@@ -0,0 +1,18 @@
+## About
+
+This applications is meant to determine RTT_MIN_VALUE for a specific BOARD.
```

Comment:
What is `RTT_MIN_VALUE` used for? What's it's meaning?

Reply:
I want to change it to `RTT_MIN_OFFSET` https://github.com/RIOT-OS/RIOT/pull/14259, currently this value is only used in `ztimer` to avoid setting callbacks that wont be triggered.

Reply:
Better add a comment.